src/test/java/com/uber/cadence/client/WorkflowOptionsTest.java

```diff
@@ -188,4 +196,10 @@ public void testInvalidCronScheduleAnnotation() throws NoSuchMethodException {
     memo.put("testKey", "testObject");
     return memo;
   }
+
+  private Map<String, Object> getTestSearchAttributes() {
+    Map<String, Object> searchAttr = new HashMap<>();
+    searchAttr.put("CustomKeywordField", "testKey");
```

Comment:
May be more interesting type than string?

Reply:
Added more types